launchpad-to-stargate/tutorial/02-module-migration.md

```diff
@@ -683,6 +683,6 @@ confirm transaction before signing and broadcasting [y/N]: y
 
 ## Migrating your own application
 
-Throughout this tutorial, we have shown you how to migrate your application from Launchpad to Stargate with the help of Starport, which will always generate the most up-to-date Cosmos SDK boilerplate code. One might be expectd to migrate their application manually by following the documentation, which is still a viable options, albeit less efficient. 
+Throughout this tutorial, we have shown you how to migrate your application from v0.39 to v0.40+ Stargate with the help of Starport, which will always generate the most up-to-date Cosmos SDK boilerplate code. One might be expectd to migrate their application manually by following the documentation, which is still a viable options, albeit less efficient. 
 
 The approach we used in this tutorial minimizes the changes we had to make when migrating the application - we only had to migrate the custom logic that was implemented, rather than modifying each individual file that needed to be updated. Having boilerplate code also allows us to reference it as an example when examining the fundamental changes within the application itself when making the necessary changes.
```

Comment:
```suggestion
Although it is possible to manually change your application code, this tutorial minimizes the changes required to migrate an application. Instead of manually modifying each file that required updates, you migrated only the custom logic that was implemented. The generated boilerplate code also serves as a useful reference to examine the fundamental changes within the application.
```